qiskit/providers/ibmq/experiment/experiment.py

```diff
@@ -140,6 +159,27 @@ def project(self) -> str:
         """Return the experiment's project."""
         return self._project
 
+    @property
+    def share_level(self) -> ExperimentShareLevel:
```

Comment:
```suggestion
    def share_level(self) -> Optional[ExperimentShareLevel]:
```
Same as before, needs `Optional` if it can be `None`.